### ISSUE TITLE
An auto generated report from spyder.

### ISSUE THREAD

**Issue by prnvjb** [spyder-ide/spyder#7358]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/usr/lib/python3/dist-packages/spyder_profiler/widgets/profilergui.py", line 264, in <lambda>
    self.finished(ec, es))
  File "/usr/lib/python3/dist-packages/spyder_profiler/widgets/profilergui.py", line 329, in finished
    self.show_data(justanalyzed=True)
  File "/usr/lib/python3/dist-packages/spyder_profiler/widgets/profilergui.py", line 350, in show_data
    self.datatree.load_data(self.DATAPATH)
  File "/usr/lib/python3/dist-packages/spyder_profiler/widgets/profilergui.py", line 462, in load_data
    stats_indi = [pstats.Stats(profdatafile),]
  File "/usr/lib/python3.6/pstats.py", line 72, in __init__
    self.init(arg)
  File "/usr/lib/python3.6/pstats.py", line 86, in init
    self.load_stats(arg)
  File "/usr/lib/python3.6/pstats.py", line 99, in load_stats
    with open(arg, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/chakri/.config/spyder-py3/profiler.results'

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 3.6.5
* Qt Versions: 5.9.5, PyQt5 5.10.1 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.22.0 (OK)
numpy >=1.7      :  1.14.5 (OK)
sphinx >=0.6.6   :  1.6.7 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
psutil >=0.3     :  5.4.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  None (NOK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  5.5.0 (OK)
pylint >=0.25    :  1.8.3 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.8). Please update by opening a terminal (e.g. xterm) and running there

    conda update spyder

**Comment by ccordoba12** [comment]
Or in this case:

    pip install -U spyder
